### ISSUE TITLE
ibus-daemon's Preferences window does not appear

### ISSUE THREAD

**Issue by cstrahan** [NixOS/nixpkgs#31454]
## Issue description

`ibus-daemon`'s "Preferences" window does not appear.

### Steps to reproduce

Start `ibus-daemon`. Right click the system tray icon, click "Preferences". Nothing appears.

This started somewhere between e0771a241092deb0a09341607adf0f24c749c037 (works) and ba22ad45fef4cef2c180895ce1af4333d286be16 (doesn't work).

This is on a plain X server + XMonad setup.

## Technical details

* System: NixOS 18.03.git.cfafd6f (Impala)
* Nix version: 1.12pre5663_c7af84ce
* Nixpkgs version: 17.09 release
* Sandboxing enabled: yes

**Comment by cstrahan** [comment]
Here's the problem, directly caused by 3378dbbeed1e69050308a4c45dd9eecc5b92567d:

```
[pid 28665] execve("/nix/store/w3ak8w2w0fv1hmcjx1l8g6r1rg4lvpbh-ibus-1.5.16/bin/ibus-setup", ["/nix/store/w3ak8w2w0fv1hmcjx1l8g"..., "ibus-setup"], 0x7ffdcf8b7e08 /* 97 vars */) = -1 ENOENT (No such file or directory)
[pid 28665] +++ exited with 1 +++
```

`ibus-daemon` needs to invoke `ibus-setup` when the user wants to configure `ibus`, so I don't see the point in removing it from the `ibus` closure: if `ibus` can't find `ibus-setup` one way or another, `ibus` doesn't function correctly.

/cc @edolstra

**Comment by yegortimoshenko** [comment]
Try #31094.

**Comment by laMudri** [comment]
Is this fixed? It's working fine on my machine.

**Comment by yegortimoshenko** [comment]
Resolved in #31094.
